pkg/proxy/metaproxier/meta_proxier.go

```diff
@@ -94,7 +94,7 @@ func (proxier *metaProxier) OnServiceSynced() {
 func (proxier *metaProxier) OnEndpointsAdd(endpoints *v1.Endpoints) {
 	ipFamily, err := endpointsIPFamily(endpoints)
 	if err != nil {
-		klog.V(4).Infof("failed to add endpoints %s/%s with error %v", endpoints.ObjectMeta.Namespace, endpoints.ObjectMeta.Name, err)
+		klog.ErrorS(nil, "failed to add endpoints", "endpoints", klog.KObj(&endpoints.ObjectMeta), "error", err)
```

Comment:
but we have an error arg - why not use it?

Reply:
oh. Sorry, I just ignore these and just focus on the other one.🙈 Have fixed it.